src/components/panels/AddProposalPanel.js

```diff
@@ -330,6 +330,7 @@ const AddProposalPanel = React.memo(({ onSubmit }) => {
           mode={neededThreshold ? 'info' : 'warning'}
           css={`
             margin-top: ${2 * GU}px;
+            margin-bottom: ${6 * GU}px;
```

Comment:
Let's set this margin to the `form` parent element, because at the bottom we have another warning component. Also lets set it to `${7 * GU}px` so the bottom spacing is even with the form upper spacing.

Reply:
Done!